app/views/homes/show.html.erb

```diff
@@ -1,2 +1,4 @@
-<h1>Welcome to Tutorify</h1>
-
+<div class="container">
+	<%= image_tag("homes/landing_image.jpg", :class => "landing-image") %>
+	<div class="topright"><h1>Welcome to Tutorify</h1></div>
```

Comment:
you don't need to define custom alignment classes, Bootstrap takes care of all that!